test-plans/WhitehallPublishingCollections.scala

```diff
@@ -75,24 +76,26 @@ class WhitehallPublishingCollections extends Simulation {
       )
       // Whitehall document searches are hard-limited to 10 results per search so use a search suffix
       // to match on the random Int used when generating the publication title.
-      .foreach((1 to 5).toList, "suffix", "index"){
-        exec(
+      .repeat(documentSearches, "index"){
+        exec(session => {
+          val suffix = session("index").as[Int] + 1
+          println(s"Searching for whitehall documents with title: Gatling test publication $suffix")
```

Comment:
Should this `println` (and the one below) still be here, or are these leftover debugging code?

Reply:
These were intentional, though I'm not convinced they add much value, they should probably be logging statements. I'll remove them.

Reply:
@barrucadu I've removed the `println`s